manifests/server.pp

```diff
@@ -166,6 +183,8 @@
   Variant[Stdlib::Port::Unprivileged,Enum['unix']] $management_port = 7505,
   String $up                                                        = '',
   String $down                                                      = '',
+  String $client_connect                                            = '',
```

Comment:
please update both variable to default to `Optional[String[1]]`. The default value should always be `undef` for strings. This also makes the handling within the template easier.

Reply:
Is there any reason it would be different than the existing `$up` and `$down` which default to `''`?